native/cocos/renderer/pipeline/PipelineSceneData.h

```diff
@@ -126,6 +128,7 @@ class CC_DLL PipelineSceneData : public RefCounted {
     CSMLayers *_csmLayers{nullptr};
     bool _isHDR{true};
     float _shadingScale{1.0F};
+    bool _isSupportCSM{true};
```

Comment:
Put _isHDR after _isSupportCSM for better struct padding and save memory.

Reply:
sure